src/modules/preview-site/components/preview-site-row.tsx

```diff
@@ -142,18 +159,18 @@ export function PreviewSiteRow( {
 								{ __( 'Updating' ) }
 							</div>
 						) : (
-							<Tooltip text={ dateString } disabled={ ! date }>
+							<Tooltip text={ dateString } disabled={ ! date || isSiteInactive }>
 								{ getLastUpdateTimeText() }
 							</Tooltip>
 						) }
 					</div>
 					<div className="flex items-center">
-						<Tooltip text={ expireDateString } disabled={ isExpired }>
+						<Tooltip text={ expireDateString } disabled={ isSiteInactive }>
 							<div className="w-[150px] text-a8c-gray-700 pl-4">{ countDown }</div>
```

Comment:
Can we render `Deleted` on this row if the site has been deleted?

Reply:
It makes sense to me. I have included it; here is the final look. 

<img width="1604" height="688" alt="Image" src="https://github.com/user-attachments/assets/56cb1539-c851-40c9-bac5-54431bf5fd70" />